src/core_plugins/elasticsearch/lib/create_admin_cluster.js

```diff
@@ -0,0 +1,19 @@
+export default function (server) {
+  const config = server.config();
+  const Logger = server.plugins.elasticsearch.ElasticsearchClientLogging;
+
+  class AdminClientLogging extends Logger {
```

Comment:
If you export this separately rather than burying the definition in this function, you can unit test AdminClientLogging as well.